StripeFinancialConnections/StripeFinancialConnections/Source/Helpers/UIColor+Extensions.swift

```diff
@@ -71,16 +79,20 @@ extension UIColor {
         return UIColor(red: 106 / 255.0, green: 115 / 255.0, blue: 131 / 255.0, alpha: 1)  // #6a7383
     }
 
+    private static var neutral600: UIColor {
+        return UIColor(red: 89 / 255.0, green: 97 / 255.0, blue: 113 / 255.0, alpha: 1)  // #596171
+    }
+
     private static var neutral800: UIColor {
-        return UIColor(red: 48 / 255.0, green: 49 / 255.0, blue: 61 / 255.0, alpha: 1)  // #30313d
+        return UIColor(red: 53 / 255.0, green: 58 / 255.0, blue: 68 / 255.0, alpha: 1)  // #353a44
     }
 
     static var brand100: UIColor {
         return UIColor(red: 242 / 255.0, green: 235 / 255.0, blue: 255 / 255.0, alpha: 1)  // #f2ebff
     }
 
     private static var brand500: UIColor {
-        return UIColor(red: 99 / 255.0, green: 91 / 255.0, blue: 255 / 255.0, alpha: 1)  // #635bff
+        return UIColor(red: 103 / 255.0, green: 93 / 255.0, blue: 255 / 255.0, alpha: 1)  // #675DFF
```

Comment:
others seems lower casesd, should this be lower case too?

Reply:
good attention to detail 💯 I'll change to lowercase